test/unit/test.xpi.js

```diff
@@ -52,6 +53,26 @@ describe("lib/xpi", function() {
     })
     .catch(done);
   });
+
+  it("Zips up cwd's addon in the specified directory", function(done) {
```

Comment:
if you remove `done` from the argument list then you can simply return a promise. You won't need to call `done()` and you won't need to add a `catch()` handler.